pymc_marketing/clv/models/basic.py

```diff
@@ -164,6 +169,20 @@ def _fit_DEMZ(self, **kwargs) -> az.InferenceData:
         with self.model:
             return pm.sample(step=pm.DEMetropolisZ(), **sampler_config)
 
+    def _fit_advi(self, **kwargs) -> az.InferenceData:
+        """Fit a model with ADVI."""
+        sampler_config = {}
+        if self.sampler_config is not None:
+            sampler_config = self.sampler_config.copy()
+        sampler_config.update(**kwargs)
+        if sampler_config.get("method") is not None:
```

Comment:
NIT: More conventional check
```suggestion
        if "method" in sampler_config:
```